Google.PowerShell/PubSub/GcpsSubscription.cs

```diff
@@ -153,4 +153,166 @@ protected override void ProcessRecord()
             }
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Retrieves one or more Google Cloud PubSub subscriptions.
+    /// </para>
+    /// <para type="description">
+    /// Retrieves one or more Google Cloud PubSub subscriptions. The cmdlet will search for subscriptions
+    /// in the default project if -Project is not used. If -Topic is used, the cmdlet will only return
+    /// subscriptions belonging to the specified topic. If -Subscription is used, the cmdlet will only return
+    /// subscriptions whose names match the subscriptions' names provided.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> Get-GcpsSubscription</code>
+    ///   <para> This command retrieves all subscriptions in the default project.</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Get-GcpsSubscription -Topic "my-topic" -Project "my-project"</code>
+    ///   <para> This command retrieves all subscriptions that belong to topic "my-topic" in the project "my-project".</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Get-GcpsSubscription -Subscription "subscription1", "subscription2"</code>
+    ///   <para> This command retrieves subscriptions "subscription1" and "subscription2" in the default project.</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Get-GcpsSubscription -Subscription "subscription1", "subscription2" -Topic "my-topic"</code>
+    ///   <para> This command retrieves subscriptions "subscription1" and "subscription2" in the topic "my-topic".</para>
+    /// </example>
+    /// <para type="link" uri="(https://cloud.google.com/pubsub/docs/subscriber#overview-of-subscriptions)">
+    /// [Subscription]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "GcpsSubscription")]
+    public class GetGcpsSubscription : GcpsCmdlet
+    {
+        /// <summary>
+        /// <para type="description">
+        /// The project to check for subscriptions. If not set via PowerShell parameter processing, will
+        /// default to the Cloud SDK's DefaultProject property.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        [ConfigPropertyName(CloudSdkSettings.CommonProperties.Project)]
+        public string Project { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// The names of the subscriptions to be retrieved.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false, Position = 0)]
+        [ValidateNotNullOrEmpty]
+        [Alias("Name")]
+        public string[] Subscription { get; set; }
```

Comment:
Yes, this is due to convention https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx